docs/resources/Guild_Scheduled_Event.md

```diff
@@ -6,54 +6,99 @@ A representation of a scheduled event in a [guild](#DOCS_RESOURCES_GUILD/).
+
+\* `creator_id` will be null and `creator` will not be included for events created before October 25th, 2021, when the concept of `creator_id` was introduced and tracked.
+
+\** See [field requirements by entity type](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-field-requirements-by-entity-type) to understand the relationship between `entity_type` and the following fields: `channel_id`, `entity_metadata`, and `scheduled_end_time` 
 
 ###### Guild Scheduled Event Privacy Level
 
 | Level      | Value | Description                                              |
 | ---------- | ----- | -------------------------------------------------------- |
-| PUBLIC     | 1     | the scheduled event is public and available in discovery |
 | GUILD_ONLY | 2     | the scheduled event is only accessible to guild members  |
 
 ###### Guild Scheduled Event Entity Types
 
 | Type           | Value |
 | -------------- | ----- |
-| NONE           | 0     |
 | STAGE_INSTANCE | 1     |
 | VOICE          | 2     |
 | EXTERNAL       | 3     |
 
+###### Field Requirements By Entity Type
+
+The following matrix describes field requirements based on current entity type.
+
+`non-null` : This field is required to be a non-null value
+
+`null`     : This field is required to be null
+
+`-`        : No strict requirements
+
+| Entity Type    | channel_id | entity_metadata | scheduled_end_time |
+| -------------- | ---------- | --------------- | ------------------ |
+| STAGE_INSTANCE |  non-null  |      null       |         -          |
+| VOICE          |  non-null  |      null       |         -          |
+| EXTERNAL       |    null    |    non-null *   |      non-null      |
+
+\* `entity_metadata` with a non-null `location` must be provided
+
+
 ###### Guild Scheduled Event Status
 
-| Type      | Value |
-| --------- | ----- |
-| SCHEDULED | 1     |
-| ACTIVE    | 2     |
-| COMPLETED | 3     |
-| CANCELED  | 4     |
+| Type        | Value |
+| ----------- | ----- |
+| SCHEDULED   | 1     |
+| ACTIVE      | 2     |
+| COMPLETED * | 3     |
+| CANCELED  * | 4     |
+
+\* Once `status` is set to `COMPLETED` or `CANCELED`, the `status` can no longer be updated
+
+###### Valid Guild Scheduled Event Status Transitions
+
+SCHEDULED --> ACTIVE
+
+ACTIVE --------> COMPLETED
+
+SCHEDULED --> CANCELED
+
 
 ###### Guild Scheduled Event Entity Metadata
 
-| Field        | Type                | Description                               |
-| ------------ | ------------------- | ----------------------------------------- |
-| location?    | string              | location of the event (1-100 characters)  |
+| Field        | Type                | Description                              |
+| ------------ | ------------------- | ---------------------------------------- |
+| location? *  | string              | location of the event (1-100 characters) |
```

Comment:
NOTE: the deprecated `speaker_ids` field was included here in the initial publication and was removed in a previous change